agimus_controller_ros/parameters.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+import rospy
+import numpy as np
+
+
+class OCPParameters:
```

Comment:
Usually what we do for parameters is that we have a class of parameters inside the core library (`agimus_controller` here). And then you add external readers for it.
In the core library you could add a file parser using a yaml file or a dictionary interface.
And in the ROS library you can add a rosparam parser.

Reply:
that's what I'm doing in fact

Reply:
but you declare your class in the ros api not ins the core api.
And in the ROS api you could simply have a class OCPParametersRos:
```python
class OCPParametersRos: OCPParameters

   def parseFromRos(self):
      pass
```

Reply:
sure we could have this kind of api, but I think this PR has already enough changes, that's for another PR